src/webpack/webpackTools.ts

```diff
@@ -38,6 +38,9 @@ export function createConfig(outPath: string, entryPoint: string, options: Custo
         optimization: {minimize: !options || options.minify !== false},
         output: {path: outPath, filename: `${options && options.outputFilename || '[name]-bundle'}.js`},
         resolve: {extensions: ['.ts', '.tsx', '.js']},
+        node: {
+            fs: 'empty'
+        },
```

Comment:
This is needed for the above check on if we are in a window or node context.  Webpack will try and bundle `fs` but because it is node it flags an error of not found.  We are ok to set it as empty as `fs` will never be used in a window context anyway.

Documentation: https://webpack.js.org/configuration/node/

Reply:
Worth a short comment in the file on this?

Reply:
And what specifically causes Webpack to try to bundle `fs`? The reference to `window`?

Reply:
I will comment the file.  Webpack moves it in because there is an import to `getProjectConfig()` which has `fs` in it.  The import itself is guarded after the window check but webpack doesnt care.

Reply:
Its here https://github.com/HadoukenIO/service-tooling/pull/47/files#diff-6eace077d503669ac830dbb43712a18dR214

Reply:
Fixed with a comment.